src/foundry/client/ui/context.d.ts

```diff
@@ -133,4 +154,27 @@ declare namespace ContextMenu {
      */
     animate?: boolean;
   }
+
+  interface ContextMenuCallback extends Function {
+    /**
+     * The element that the context menu has been triggered for.
+     */
+    target: HTMLElement;
+  }
+
+  interface ConstructorOptions {
+    /**
+     * The name of the hook to call
+     * @defaultValue `EntryContext`
+     */
+    hookName?: string;
+    /**
+     * A function to call when the context menu is opened.
+     */
+    onOpen?: ContextMenuCallback;
+    /**
+     * A function to call when the context menu is closed.
+     */
+    onClose?: ContextMenuCallback;
```

Comment:
nit, usually when we have fields with documented properties we usually add spaces between them like so:
```suggestion
    /**
     * The name of the hook to call
     * @defaultValue `EntryContext`
     */
    hookName?: string;

    /**
     * A function to call when the context menu is opened.
     */
    onOpen?: ContextMenuCallback;

    /**
     * A function to call when the context menu is closed.
     */
    onClose?: ContextMenuCallback;
```